Source/WebKit/Scripts/webkit/tests/TestSerializedType.serialization.in

```diff
@@ -291,3 +290,19 @@ header: "RemoteVideoFrameIdentifier.h"
     IPC::ObjectIdentifierReference<WebKit::RemoteVideoFrameIdentifier> reference();
     uint64_t pendingReads();
 };
+
+#if ENABLE(OUTER_CONDITION)
+[Nested] class Namespace::OuterClass {
+    int outerValue;
+};
+
+#if ENABLE(INNER_CONDITION)
+enum class EnumNamespace::InnerEnumType : uint8_t {
+    InnerValue,
+#if ENABLE(INNER_INNER_CONDITION)
+    InnerInnerValue,
+#endif
+}
+#endif
```

Comment:
could you add an else here?  That's worth testing.

Reply:
Done!